metadata/index.html

```diff
@@ -1678,7 +1696,10 @@
           Conversion specifications MUST define a default mapping from an annotated table that lacks any annotations (ie that is equivalent to an un-annotated table).
         </p>
         <p>
-          Conversion specifications MUST use either the <a href="#column-name"><code>name</code></a> or the <a href="#column-predicateUrl"><code>predicateUrl</code></a> of a column as the basis for naming machine-readable fields in the target format, such as the name of the equivalent element or attribute in XML, property in JSON or property URI in RDF.
+          Conversion specifications MUST use the <a>property value</a> of the <a href="#cell-aboutUrl"><code>aboutUrl</code></a> of the first cell column, if any, as the basis for identifying a mapped row.
+        </p>
+        <p>
+          Conversion specifications MUST use the <a>property value</a> of the <a href="#cell-propertyUrl"><code>propertyUrl</code></a> of a column as the basis for naming machine-readable fields in the target format, such as the name of the equivalent element or attribute in XML, property in JSON or property URI in RDF.
```

Comment:
Add a 'if provided'. It's reasonable to use the `name` of the column if it's not provided and we shouldn't expect that properties will always be identified as URLs (it's unlikely to be useful to do that for the JSON conversion, for example).